### ISSUE TITLE
[Accessibility] [Bug] - The side-navigation list items lack proper structure/ARIA-assigned roles

### ISSUE THREAD

**Issue by MonishaK28** [SAP/fundamental-ngx#13558]
### Describe your proposal

In the side navigation panel, an LI does not have a ul element (without an ARIA-assigned role); ol element (without an ARIA-assigned role); or element with a role set to the value: list as a parent.  

The issue exist for Core/CX as well as BTP versions. 

<img width="940" height="337" alt="Image" src="https://github.com/user-attachments/assets/17461261-0cf0-4511-8746-a6ce53bb3d98" />

### Which versions of Angular and Fundamental Library for Angular are affected?

Fundamental Lib- V0.56.4

### Declaration

- [x] I'm not disclosing any internal or sensitive information.

**Comment by droshev** [comment]
@MonishaK28 Did you try with this version of side nav: https://fundamental-ngx.netlify.app/#/btp/navigation?

**Comment by MonishaK28** [comment]
> [@MonishaK28](https://github.com/MonishaK28) Did you try with this version of side nav: https://fundamental-ngx.netlify.app/#/btp/navigation?

Yes, the issue persists here as well. 

<img width="940" height="364" alt="Image" src="https://github.com/user-attachments/assets/08c85347-9e5d-42e9-987d-cc0269d1ccaa" />
